common/scripts/install-run-rush.js

```diff
@@ -0,0 +1,86 @@
+"use strict";
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See the @microsoft/rush package's LICENSE file for license information.
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+    if (k2 === undefined) k2 = k;
```

Comment:
Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/gh/matteobruni/tsparticles/pullRequest?prid=6397531)